packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -41,19 +30,9 @@ contract LockedGold is ILockedGold, ReentrancyGuard, Initializable, UsingRegistr
     PendingWithdrawal[] pendingWithdrawals;
   }
 
-  struct Account {
-    bool exists;
-    // Each account may authorize additional keys to use for voting or valdiating.
-    // These keys may not be keys of other accounts, and may not be authorized by any other
-    // account for any purpose.
-    Authorizations authorizations;
-    Balances balances;
-  }
-
-  mapping(address => Account) private accounts;
+  mapping(address => Balances) private accounts;
```

Comment:
This is probably better named as `balances` to avoid confusion with `Accounts.sol`?

Reply:
Updated!